templates/gitlab/mender-artifact-upload.gitlab-ci.yml

```
@@ -0,0 +1,26 @@
+# GitLab CI template to upload Mender artifacts from MENDER_ARTIFACTS_FOLDER folder
+# to a Mender server.
+#
+# The template is designed to be used with the includes keyword or copied into an existing
+# configuration. It requires `MENDER_SERVER_URL` and `MENDER_SERVER_ACCESS_TOKEN` variables
+# to work and can be used to upload a Mender artifacts from `MENDER_ARTIFACTS_FOLDER` folder
+# to a Mender server.
+
+include:
+  - remote: "https://raw.githubusercontent.com/mendersoftware/mender-ci-workflows/master/templates/gitlab/common.gitlab-ci.yml"
+
+.upload:mender:artifact:
+  image: mendersoftware/mender-ci-tools:master
+  script:
+    - for ARTIFACT in $(ls ${MENDER_ARTIFACTS_FOLDER}/*.mender); do
```

`.mender` suffix is just a convention, it could really be anything so is best to remove
```suggestion
    - for ARTIFACT in $(ls ${MENDER_ARTIFACTS_FOLDER}/*); do
```